test/jest.e2e.setup.ts

```diff
@@ -1,14 +1,20 @@
 // Global setup is performed before a TypeScript environment is made available, so we need to create the environment manually
 require('ts-node/register')
 
+import crypto from 'crypto'
 import { setup as setupDevServer } from 'jest-dev-server'
 import path from 'path'
 import { v4 as uuidv4 } from 'uuid'
 
 const setup = async () => {
   process.env.SKIP_LOAD_ENV = 'true'
   process.env.ENABLE_EXPERIMENTAL_SOCKETS = 'true'
-  process.env.DATABASE_URL = path.join(__dirname, '.test-data', `${uuidv4()}.sqlite`)
+  process.env.DATABASE_URL = process.env.DATABASE_URL || path.join(__dirname, '.test-data', `${uuidv4()}.sqlite`)
+
+  if (process.env.DATABASE_URL.startsWith('postgres')) {
+    process.env.DATABASE_SUFFIX = `__${randomLetters(8)}`
+    process.env.DATABASE_TRANSIENT = 'true'
+  }
```

Comment:
DATABASE_TRANSIENT doesn't work for e2e tests because jest doesn't let the server shut down completely. We'll need to look into a way of letting the dev server shutdown completely before killing the process. The database connection even stays alive.

Reply:
Fixed